torch/csrc/cudnn/Conv.cpp

```diff
@@ -225,6 +225,14 @@ void cudnn_convolution_forward(
   size_t workspaceSize;
   CHECK(cudnnGetConvolutionForwardWorkspaceSize(handle, idesc.desc, wdesc.desc,
       cdesc.desc, odesc.desc, fwdAlg, &workspaceSize));
+  size_t memFree;
+  size_t memTotal;
+  cudaMemGetInfo(&memFree, &memTotal);
```

Comment:
@qunluo Could you point out where in the code the out of memory error was issued?
Normally we should be fine whenever we call `THCudaMalloc`, and [`Workspace` calls it to allocate space](https://github.com/pytorch/pytorch/blob/master/torch/csrc/cudnn/Conv.cpp#L90).